### ISSUE TITLE
Python: __init__ with -> None: at end

### ISSUE THREAD

**Issue by rwols** [20Tauri/DoxyDoxygen#140]
The following code:

```python
class DottedDict:

    __slots__ = ('_d')

    def __init__(self, d: Optional[Dict[str, Any]] = None) -> None:
        self._d = {}  # type: Dict[str, Any]
        if d is not None:
            self.update(d)
```

Produces the following template when press Alt+Q on the `__init__` token:

```python
class DottedDict:

    __slots__ = ('_d')

    def __init__(self, d: Optional[Dict[str, Any]] = None)
    """
    Constructs a new instance.
    
    :param      d:    { parameter_description }
    :type       d:    { type_description }
    """ -> None:
        self._d = {}  # type: Dict[str, Any]
        if d is not None:
            self.update(d)
```
It should instead create this template:
```python
class DottedDict:

    __slots__ = ('_d')

    def __init__(self, d: Optional[Dict[str, Any]] = None) -> None:
        """
        Constructs a new instance.
        
        :param      d:    { parameter_description }
        :type       d:    { type_description }
        """
        self._d = {}  # type: Dict[str, Any]
        if d is not None:
            self.update(d)
```

**Comment by 20Tauri** [comment]
Thanks for reporting,

Fixed in 0.76.2 (just released in both evolution and regular versions)

**Comment by rwols** [comment]
Thanks!
